main.go

```diff
@@ -1,107 +1 @@
-package main
 
-import (
-	"database/sql"
-	"fmt"
-	"log"
-	"os"
-	
-
-	_ "github.com/mattn/go-sqlite3"
-)
-
-func openDB(dbfile string) (*sql.DB) {
-	db, err := sql.Open("sqlite3", dbfile)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return(db)
-}
-
-func getHandleID(db *sql.DB, handle string) ([]int){
-	var handleID []int
-
-	rows, err := db.Query("SELECT rowid, id, service FROM handle where id=?", handle )
-	// fmt.Println("SELECT rowid, id, service FROM handle where id='%s'", handle)
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer 	rows.Close()
-
-	for rows.Next() {
-		var rowid int
-		var id string
-		var service string
-		err = rows.Scan(&rowid, &id, &service)
-		if err != nil {
-			log.Fatal(err)
-		}
-		handleID = append(handleID, rowid)
-	}
-	err = rows.Err()
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return handleID
-}
-
-func getMessages(db *sql.DB, handles []int, name string) {
```

Comment:
consider instead of passing in the db to this function making a struct that you initialize with the db then this function being a pointer receiver of that struct